Ports/Android/src/com/codename1/impl/android/AndroidImplementation.java

```diff
@@ -4235,6 +4235,36 @@ public boolean onTouch(View v, MotionEvent me) {
                                     return myView.getAndroidView().onTouchEvent(me);
                                 }
                             });
+                            /**
+                             * Mouse Event Listener used to handle mouse wheel events
+                             * The mouse wheel can be used to scroll the page up or down
+                             */
+                            layoutWrapper.setOnGenericMotionListener(new View.OnGenericMotionListener() {

```

Comment:
This adds a motion listener to a native peer component. E.g. a webview or other native component.  Not the main view.  Probably we need to add similar in CodenameOneView if we want this to be for general app.